### ISSUE TITLE
Update to Asp.Net Core 3.1

### ISSUE THREAD

**Issue by Chibuikekenneth** [nbarbettini/little-aspnetcore-book#78]
Hy, 
This book is great and still helpful for those learning Asp.net Core and intro to Web programming, have you considered updating the `dotnet` version as well as its dependencies used in this book?

**Comment by nbarbettini** [comment]
Yep! An update to 3.1 is badly needed. I've been delayed by a new baby 🙂 but it's still planned. I'm not sure yet when it will be ready, stay tuned.

**Comment by arunpratap26** [comment]
congratulations  take your time
